addons/website_event/static/src/js/website_event.editor.js

```diff
@@ -2,11 +2,237 @@ odoo.define('website_event.editor', function (require) {
+                            id: _.uniqueId('address_'),
+                            create: true,
+                            text: _.str.sprintf(_t('Create "%s"'), name),
+                            name: name,
+                        };
+                    }
+                },
+                fill_data: function (query, data) {
+                    var that = this;
+                    var partners = {results: []};
+                    _.each(data, function (obj) {
+                        var contact_address = obj.contact_address.replace(/[\n]+/g, '').trim();
+                        if (contact_address && that.matcher(query.term, contact_address)) {
+                            partners.results.push({id: obj.id, text: contact_address});
+                        }
+                    });
+                    query.callback(partners);
+                },
+                query: function (query) {
+                    var that = this;
+                    // fetch data only once and store it
+                    if (!this.selection_data) {
+                        self._rpc({
+                            model: 'res.partner',
+                            method: 'search_read',
+                            fields: ['contact_address']
+                        }).then(function (data) {
+                            that.fill_data(query, data);
+                            that.selection_data = data;
+                        });
+                    } else {
+                        this.fill_data(query, this.selection_data);
+                    }
+                }
+            });
+        });
+    },
+    /**
+     * @param {*} errors
+     */
+    _display_errors: function (errors) {
+        this.$("p.text-danger").toggleClass('d-none', true);
+        for (let i = 0; i < errors.length; i++) {
+            this.$("#" + errors[i] + "_error").toggleClass('d-none', false);
+        }
+    },
+    _getSelect2DropdownValues: function () {
+        var select2Value = this.$('#address_ids').select2('data');
+        if (select2Value){
+            if (select2Value.create) {
+                return [0, {'name': select2Value.name}];
+            } else {
+                return [select2Value.id];
+            }
+        }
+        return [];
+    },
+    /**
+     *
+     * @param {*} $dateGroup
+     */
+    _initDateRangePicker: function ($dateGroup) {
+        var minDate = moment().subtract(1, "d");
+        var maxDate = moment().add(200, "y");
+        var self = this;
+
+        $dateGroup.daterangepicker({
+            minDate: minDate,
+            maxDate: maxDate,
+            startDate: this.eventStart,
+            endDate: this.eventEnd,
+            opens: 'left',
+            viewDate: moment(new Date()).hours(minDate.hours()).minutes(minDate.minutes()).seconds(minDate.seconds()).milliseconds(minDate.milliseconds()),
+            timePicker: true,
+            timePicker24Hour: true,
+            locale: {
```

Comment:
Shoudln't that be localization-based ? Like I suppose not everyone uses ltr display ?